src/App.jsx

```diff
@@ -1,5 +1,22 @@
+import { useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import ListContainer from './ListContainer';
```

Comment:
외부에서 불러오는 모듈과, 컴포넌트 사이에는 한줄 띄어주세요 :)
```suggestion
import { useDispatch } from 'react-redux';
import { useEffect } from 'react';

import ListContainer from './ListContainer';
```